File: customview-size-layout/src/main/java/com/zhaofan/customview_size_layout/MainActivity.java
package com.zhaofan.customview_size_layout;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onSquareImageClick(View view){
        startActivity(new Intent(this,SquareImageActivity.class));
    }

    public void onSimpleCircleClick(View view){
        startActivity(new Intent(this,SimpleCircleActivity.class));
    }

    public void onTagLayoutClick(View view){
        startActivity(new Intent(this,TagLayoutActivity.class));
    }
}

File: app/src/main/java/com/zhaofan/ptrandroidstu/MainActivity.kt
package com.zhaofan.ptrandroidstu

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }


    fun onMaterialEditTextDemoClick(view:View){
        startActivity(Intent(this,MaterialEditTextActivity::class.java))
    }
}
